graphql/resolvers.go

```diff
@@ -1937,6 +1937,9 @@ func (r *mutationResolver) AddAnnotationIssue(ctx context.Context, taskID string
 	apiIssue restModel.APIIssueLink, isIssue bool) (bool, error) {
 	usr := MustHaveUser(ctx)
 	issue := restModel.APIIssueLinkToService(apiIssue)
+	if err := annotations.ValidateIssueURL(issue.URL); err != nil {
```

Comment:
This looks great! Did you test it in the playground with a malformed url?

Reply:
Yup, I tested it using the sample mutation you showed me! Changing the URL to an invalid one like the ones in the tests would result in errors.

Reply:
sweet!